metadata/org.ehcache/ehcache/3.10.8-jakarta/resource-config.json

```diff
@@ -0,0 +1,64 @@
+{
+  "bundles": [
+    {
+      "name": "com.sun.org.apache.xerces.internal.impl.msg.SAXMessages",
+      "locales": [
+        "und",
```

Comment:
Could you please elaborate on why only those locales are needed?
I am also wondering if that belongs in ehcache hints or not.

@dnestoro @gradinac Do we support conditions on bundles?

Reply:
- This is simply generated by Conditional Agent Mode. Do I need to remove it?